test/data/templates/old-default-anki-field-templates.handlebars

```diff
@@ -28,6 +28,10 @@
     {{~definition.dictionary~}}
 {{/inline}}
 
+{{#*inline "dictionary-alias"}}
```

Comment:
This file shouldn't be edited, it's there just to test that applying all the handlebars updates to it results in the latest default handlebars file.